docs/application/dotnet/guides/multimedia/pose-detection.md

```diff
@@ -0,0 +1,182 @@
+LeftLowLeg,15
+LeftFoot,16
+RightUpLeg,11
+RightLowLeg,12
+RightFoot,13
+```
+If there is no mapped landmark, you don't need to list it. For example, index three is missing. It means that, in this BVH file, the third definition, `MV_INFERENCE_HUMAN_POSE_THORAX` is not defined and not used.
+
+## Prerequisites
+
+To enable your application to use the Media Vision Inference functionality:
+
+1.  Install the NuGet packages for Media Vision.
+
+    ```csharp
+    using Tizen.Multimedia;
+    using Tizen.Multimedia.Vision;
+    ```
+2. Create a structure to store pose data.
+
+   For pose detection, use the following `BodyPart` structure:
+
+   ```csharp
+   public enum BodyPart
+   {
+       Head,
+       Neck,
+       RightUpArm,
+       RightLowArm,
+       RightHand,
+       LeftUpArm,
+       LeftLowArm,
+       LeftHand,
+       RightUpLeg,
+       RightLowLeg,
+       RightFoot,
+       LeftUpLeg,
+       LeftLowLeg,
+       LeftFoot,
+   }
+   ```
+
+## Detect human pose
+
+To detect human pose from an image:
+
+1. Create the source and engine configuration handles:
+
+   ```csharp
+   pdConfig = new InferenceModelConfiguration();
+   pdConfig.WeightFilePath = Application.Current.DirectoryInfo.Resource + "model.tflite";
+   pdConfig.MetadataFilePath = Application.Current.DirectoryInfo.Resource + "model.json";
+   pdConfig.Backend = InferenceBackendType.TFLite;
+   pdConfig.Device = InferenceTargetDevice.CPU;
+   pdConfig.LoadInferenceModel();
+   ```
+
+2. Get the image file and fill the MediaVisionSource `source` with the decoded raw data.
+   In the following example image `sample.jpg`, a person is shown in a squat pose:
+
+   <img alt="sample.jpg" src="/application/native/guides/multimedia/media/mediavision_pose_sample_sumo.png" width=300>
+
+   ```csharp
+   MediaVisionSource source = new MediaVisionSource(rgbframe, width, height, Tizen.Multimedia.ColorSpace.Rgb888);
+   ```
+
+3. To detect landmarks of the pose from the `sample.jpg` image, run a `DetectAsync` from Detector.
+   This will return `Landmark`:
+
+   ```csharp
+   Task<Landmark[,]> taskLandmark = Tizen.Multimedia.Vision.PoseLandmarkDetector.DetectAsync(source, pdConfig);
+   ```
+
+4. Use `Landmark` to get landmarks information from the image:
+   Landmark is a two-dimension array that represents the number of sources for the first dimension and enum `BodyPart` for the second dimension.
+
+    ```csharp
+    public struct Landmark
+    {
+        /// <summary>
+        /// Represents a location in the 2D space.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public Point Location
+        {
+            get;
+            set;
+        }
+        /// <summary>
+        /// Confidence score of point.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public float Score
+        {
+            get;
+            set;
+        }
+    }
+    ```
+
+## Related information
+
+- Dependencies
+  - Tizen 6.5 or higher for Mobile
```

Comment:
For .Net guides we don't mention Mobile/Wearable, so kindly change to
```suggestion
  - Tizen 6.5 or higher
```

Reply:
Question. This `Tizen X.X and Higher` is used to other all pages, only in dotnet about 150 pages .

Are you going to change the style to use small letter to "higher",
and update every pages including dotnet and native and web?
If not, how about following the existing style, as guided to you with https://github.com/Samsung/tizen-docs/blob/52cdb00867da738ebd15afc4c5cce88530cb993b/reviewguide/review_points_guide.md#related_info_dependencies?

Ref history : https://github.com/Samsung/tizen-docs/pull/1567#discussion_r764821794

```
@asm-foysal asm-foysal 5 days ago
Suggested change 
  - Tizen 6.5 and Higher for Mobile
  - Tizen 6.5 or higher for Mobile
```

Reply:
@mijin85cho  capital 'Higher' makes sense. I'll commit.
but another issue is 'and' 'or' case. I guess 'Tizen 6.5 or Higher' is correct. please confirm.

Reply:
As Tizen 6.5 is included in supported version, I think 'and' is proper.
Doesn't 'or' have meaning of "only Tizen 6.5 is supported, otherwise only higher version(ex Tizen 7.0....etc except Tizen 6.5) is supported"?